createdisk.sh

```diff
@@ -29,10 +29,22 @@ function create_disk_image {
     rm -fr $tarballDirectory/${VM_PREFIX}-master-0 $tarballDirectory/${VM_PREFIX}-base
 }
 
+function update_json_description {
+    cat $1/crc-bundle-info.json \
+        | ${JQ} ".clusterInfo.masterHostname = \"${VM_PREFIX}-master-0\"" \
+        | ${JQ} '.clusterInfo.sshPrivateKeyFile = "master_privatekey"' \
```

Comment:
@cfergeau shouldn't it would be now `id_rsa_crc` ?

Reply:
I kept the initial name, but I definitely prefer id_rsa_crc :)